signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/scheduler.js

```diff
@@ -0,0 +1,98 @@
+const tracer = require('./tracer');
+
+const dbOption = require('./config').mysqlVersion;
+
+let myDB;
+
+if (dbOption == 1) {
+  myDB = require('./dbDriver/mysql_driver');
+} else {
+  myDB = require('./dbDriver/mysql2_driver');
+}
```

Comment:
I think this should be moved to `deedScheduler/dbDriver/index.js` such that the desired driver is exported instead of having this logic at the consumer level.